tests/network/vmi_networking.go

```diff
@@ -177,6 +177,13 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 					Expect(vmi.Status.Interfaces[0].MAC).To(Equal(vmi.Spec.Domain.Devices.Interfaces[0].MacAddress))
 				}
 				Expect(libnet.CheckMacAddress(vmi, "eth0", vmi.Status.Interfaces[0].MAC)).To(Succeed())
+				// run only once
+				if vmiRef == &inboundVMI {
```

Comment:
I do not consider this a valid condition in a table test.
On one hand a table test is used to take advantage of the same body given different inputs, but then we have this condition in the body which dismisses that and splits it per input.

The assertion of the vnic vendor is also unrelated to the test main subject (connectivity).
So I do not think it is a fit.

It looks a better fit to combine it with "[test_id:1770]should expose the right device type to the guest".

Reply:
I do not understand this comment.
Any reviewer or author, old and new alike, needs to reason for the changes and comments.

You should strive to improve the tests by removing redundant ones, by optimizing existing ones and unifying others. If you have a better suggestion from what is given in the review, you should raise it.

Reply:
I'll feedback on the new change, I think you could have avoided the creation of a new VM. At least, that was obvious for me when giving the suggestion.